torch/_dynamo/variables/ctx_manager.py

```diff
@@ -147,6 +147,37 @@ def fn_name(self):
         return "set_grad_enabled"
 
 
+class TorchFunctionDisableVariable(ContextWrappingVariable):
+    """represents whether torch function overrides are enabled or not"""
+
+    _guards_singleton = {
+        Guard("", GuardSource.GLOBAL, GuardBuilder.TORCH_FUNCTION_STATE)
+    }
+
+    @staticmethod
+    def create(tx, **kwargs):
+        var = TorchFunctionDisableVariable(
+            target_values=[False],
+            initial_values=[torch._C._is_torch_function_enabled()],
+            **kwargs,
+        )
+        var._call_func(tx, [False])
+        return var
```

Comment:
Why .create instead of just invoking this `_call_func` in the ctor? when would I not want this invoked in the ctor?

Reply:
I think it's like this so that you don't reinvoke it if you call clone

Reply:
Add a comment :)